src/prog_algs/metrics/toe_profile_metrics.py

```diff
@@ -85,3 +87,24 @@ def prognostic_horizon(toe_profile : ToEPredictionProfile, criteria_eqn : Callab
     # Return PH when criteria not met for at least one event key
     return ph_result
 
+def cumulative_relative_accuracy(toe_profile : ToEPredictionProfile, ground_truth : dict, **kwargs) -> Dict[str, float]:
+    """
+    Compute cumulative relative accuracy for a given profile, defined as the normalized sum of relative prediction accuracies at specific time instances.
+    
+    CRA = Σ(RA)/N for each event
+    Where Σ is summation of all relative accuracies for a given profile and N is the total count of profiles (Journal Prognostics Health Management, Saxena et al.)
+    Args:
+        toe_profile (ToEPredictionProfile): A profile of predictions, the combination of multiple predictions
+        ground_truth (dict): Dictionary containing ground truth; specified as key, value pairs for event and its value. E.g, {'event1': 47.3, 'event2': 52.1, 'event3': 46.1}
+        kwargs (optional): configuration arguments. Accepted args include:
+
+    Returns:
+        dict: Dictionary containing cumulative relative accuracy (value) for each event (key). e.g., {'event1': 12.3, 'event2': 15.1}
+    """
+    ra_sums = defaultdict(int)
+    for uncertaindata in toe_profile.values():
+        for event,value in uncertaindata.relative_accuracy(ground_truth).items():
+            ra_sums[event] += value
+    return {event:ra_sum/len(ground_truth) for event,ra_sum in ra_sums.items()}
```

Comment:
```suggestion
    return {event:ra_sum/len(toe_profile) for event, ra_sum in ra_sums.items()}
```
I think we want to divide by length of ra_sums here, since that's the structure that we're iterating over.